app/src/gecko/java/org/mozilla/tv/firefox/ext/View.kt

```diff
@@ -26,6 +30,27 @@ fun View.validateKnownViewById(): Int {
     return this.id
 }
 
+/**
+ * Set the horizontal margins on the given view.
+ *
+ * We want to add padding to the beginning and end of the RecyclerView: ideally we'd just add
+ * paddingStart/End. Unfortunately, this causes a visual glitch as each card scrolls offscreen.
+ * Instead, we set the margins for the first and last card.
+ *
+ * See [DefaultChannelAdapter.onBindViewHolder] for an example
+ */
+fun View.setHorizontalMargins(context: Context, position: Int, itemCount: Int) {
```

Comment:
At first glance, I would expect a fun called `setHorizontalMargins` to take in two ints and set them to the left and right margins for a view.  Could we rename this to better communicate that it's handling a much more specific job?  `setChannelMarginByPosition` or something?  That's super wordy, but you see the direction I'm trying to move.